rosco-web/config/rosco.yml

```diff
@@ -5,7 +5,9 @@ rosco:
   configDir: /some/path/to/config/packer
   jobs:
     local:
-      timeoutMinutes: 20
+      timeoutMinutes: 30
```

Comment:
LGTM, my one comment is currently in azure bake stage we only support use of a single template.  We have a backlog to support multiple templates which I was about to do in next week or two.  WIll include you on that PR when submitted as I assume you need that.  @scotmoor @ericjizbmsft